internal/cluster/cluster_test.go

```diff
@@ -243,7 +243,7 @@ var _ = Describe("TestClusterMonitoring", func() {
 				expectedState = models.ClusterStatusFinalizing
 			})
 			It("installing -> error", func() {
-				mockMetric.EXPECT().ClusterInstallationFinished(gomock.Any(), "error", gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
+				mockMetric.EXPECT().ClusterInstallationFinished(gomock.Any(), "error", "installing", gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).AnyTimes()
```

Comment:
Use constants

Reply:
Using constants for the states. Leaving the dashboard labels as-is so they won't be used for other uses